### ISSUE TITLE
REQUEST: Show Drag Ruler even if normal "Display Ruler Measurement" is turned off.

### ISSUE THREAD

**Issue by GambetTV** [manuelVo/foundryvtt-drag-ruler#53]
I really like the ruler showing up so we can witness the exact path being taken in this module. I hate that every time the players try to use the normal measurement tool it shows up on everyone's screens, it's super distracting and isn't something anyone needs to see.

Is there a way for you to make it so we can have both? Basically let us not show each other our normal Ruler Measurements, but still keep them when dragging the token?

**Comment by manuelVo** [comment]
Am I getting this right: You'd like to make it so players don't see each other's regular ruler, but at the same time they should be able to see each other's Drag Ruler?

**Comment by GambetTV** [comment]
Correct. I want to be able to disable people being able to see people measuring objects, but still get full functionality out of your module, and right now disabling the normal ruler (for other players being able to see it) also disables being able to see your ruler line.
